src/Wallabag/CoreBundle/Resources/config/services.xml

```diff
@@ -21,6 +21,8 @@
                  class="Wallabag\CoreBundle\Security\Firewall\WsseListener" public="false">
```

Comment:
Are you sure? :smile: http://symfony.com/doc/current/best_practices/business-logic.html#service-format-yaml